README.md

```diff
@@ -103,6 +103,23 @@ python3 -m pip install -e .
 'an hour ago'
 ```
 
+#### Also accepts iso8601 text strings
+
+```pycon
+>>> import humanize
+>>> humanize.naturaldate('2007-06-05')
+'Jun 05 2007'
+>>> humanize.naturaldate('2007-06-05T22:00:10')
+'Jun 05 2007'
+>>> humanize.naturaldate('2007-06-05T22:00:10+02:00')
+'Jun 05 2007'
+>>> humanize.naturaltime('22:00:10') # run at 22:00:00
+'10 seconds from now'
+
+Relies on the native `fromisoformat` function that exists on date, datetime and time objects.
+```
```

Comment:
Please can you move the final triple backticks after '10 seconds from now'?

They indicate where the special code formatting ends.

For example, right now it looks like:

<img width="1033" alt="image" src="https://github.com/python-humanize/humanize/assets/1324225/4a1a82f1-455b-4251-b874-a38e2adcc2ac">

https://github.com/velle/humanize/blob/iso8601/README.md#also-accepts-iso8601-text-strings